libs/api/domains/auth/src/lib/dto/apiScopes.input.ts

```diff
@@ -0,0 +1,9 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsString } from 'class-validator'
+
+@InputType('AuthApiScopesInput')
+export class ApiScopesInput {
+  @Field(() => String)
+  @IsString()
+  lang = 'is'
```

Comment:
should this be an enum?

FYI: `IsString` is not needed, doesn't do anything for the GraphQL schema validation

Reply:
Ah. Good to know. Copied the input pattern from the cms resolvers. They have an enum there but not sure if this should be an enum. 

To add a new language we "just" need to update the locale picker and add content in the backend. Enums just increase the number of code changes required :)